ros/lib/utils.py

```diff
@@ -59,3 +59,17 @@ def user_data_from_identity(identity):
         return None
 
     return identity['user']
+
+
+def sort_io_dict(performance_utilization: dict, remove_io_dict: bool = True):
+    """
+    Sorts io dict by max_io in descending order.
+    """
+    io_dict = performance_utilization['io']
+    io_dict_key = 'io_all' if remove_io_dict is True else 'io'
```

Comment:
I think you can keep key as `io_all` at first place ONLY. Please correct me.

Reply:
Yes, these changes are only applicable for the HostHistoryApi response, will revert them if not required.

Reply:
Change reverted and pushed :zap: